pkgs/development/python-modules/visions/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, attrs
+, imagehash
+, matplotlib
+, multimethod
+, networkx
+, numpy
+, pandas
+, pillow
+, pydot
+, pygraphviz
+, shapely
+}:
+
+buildPythonPackage rec {
+  pname = "visions";
+  # latest release (Dec 2021) requires barely used 1.8GB `tangled-up-in-unicode` packae
```

Comment:
How do we want to handle updates?

Reply:
```suggestion
  # latest release (Dec 2021) requires barely used 1.8GB `tangled-up-in-unicode` package
```

Reply:
Switched from unstable to release version.